Changes.md

```diff
@@ -7,6 +7,11 @@ Improvements
 - Light Editor : Added `is_sphere` column for Cycles lights.
 - Windows : Gaffer now uses the TBB memory allocator for significantly better performance.
 
+Fixes
+-----
+
+- Instancer : Added support 64 bit ints for ids ( matching what is loaded from USD ).
```

Comment:
Sorry, this needs moving again, since we've just released 1.5.0.0. Could you also retarget the PR to `1.5_maintenance` please?

Reply:
I think the grammar got a bit wonky in the change of tense too - perhaps "Added support *for* 64 bit ids"?

Reply:
Addressed.

Reply:
This would be an Improvement or Fix rather than a Feature, and in the Improvement case would be listed above the existing Fixes section.